### DISCUSSION TITLE
Creating Customizable Index List With amsthm Package

### DISCUSSION THREAD

**Discussion by Jmmo-Gomes** [joaomlourenco/novathesis#442] [Q&A]
Hi!
I want to create my own customizable index list using amsthm package for definitions and algorithms.
I ve searched online and I came across with the following [thread](https://tex.stackexchange.com/questions/263525/creating-a-list-of-theorems-divided-into-subsections-with-amsthm
) that does that for theorems.
I just wanna know if there is a better way to go about this in the novathesis template.

**Comment by joaomlourenco** [comment]
There is already support for a `\listofalgorithms`.  
Additional "\listofSOMETHING" can be added in file `Config/6_list_of.tex`.
Please share your solution / approach. Thanks.
